sound/soc/sof/ipc4-topology.c

```diff
@@ -38,6 +38,36 @@ MODULE_PARM_DESC(ipc4_ignore_cpc,
 static DEFINE_IDA(alh_group_ida);
 static DEFINE_IDA(pipeline_ida);
 
+struct sof_comp_domains {
+	const char *name;
+	enum sof_comp_domain domain;
+};
+
+static const struct sof_comp_domains sof_domains[] = {
+	{ "LL", SOF_COMP_DOMAIN_LL, },
+	{ "DP", SOF_COMP_DOMAIN_DP, }
+};
+
+static enum sof_comp_domain find_domain(const char *name)
+{
+	int i;
+
+	for (i = 0; i < ARRAY_SIZE(sof_domains); i++) {
+		if (strcmp(name, sof_domains[i].name) == 0)
+			return sof_domains[i].domain;
+	}
+
+	return SOF_COMP_DOMAIN_UNSET;
```

Comment:
add comment that LL is default if unset and return LL

Reply:
So we drop support for domain set in manifest? This is going to cause change in operation, currently we obey the manifest information and set the module to DP if it is configured to be DP, if the token is not set (which is the case for all released topology) then we effectively drop support for DP using those releases.